pandas/core/resample.py

```diff
@@ -1675,12 +1737,16 @@ def _get_period_range_edges(first, last, offset, closed="left", base=0):
         The beginning Period of the range to be adjusted.
     last : pd.Period
         The ending Period of the range to be adjusted.
-    offset : pd.DateOffset
-        The dateoffset to which the Periods will be adjusted.
+    freq : pd.DateOffset
+        The freq to which the Periods will be adjusted.
     closed : {'right', 'left'}, default None
         Which side of bin interval is closed.
-    base : int, default 0
-        The "origin" of the adjusted Periods.
+    origin : pd.Timestamp, default None
+        The timestamp on which to adjust the grouping. It must be timezone
```

Comment:
`It must be timezone aware if the index of the resampled data is.` -> `The timezone of the timestamp must match `the timezone of the index`

Reply:
Same comment: I did the change. However, I am not sure this entirely true since the `origin` just needs to be timezone aware when the index is. But I agree with you since it is less confusing for the reader anyway.